frontend/src/__tests__/cypress/cypress/pages/modelServing.ts

```diff
@@ -111,6 +111,10 @@ class InferenceServiceModal extends Modal {
     return this.findFooter().findByTestId('modal-submit-button');
   }
 
+  findCancelButton(): Cypress.Chainable<JQuery<HTMLElement>> {
+    return this.findFooter().findByTestId('modal-cancel-button');
+  }
+
```

Comment:
This is already implemented in the base `Modal` that this class extends -- why is that not working?